### DISCUSSION TITLE
Frigate, Yi Home Camera 1080p: any experience?

### DISCUSSION THREAD

**Discussion by outofsight** [roleoroleo/yi-hack-Allwinner-v2#409] [General]
I know the camera is a cheap wifi camera with low performance and far from ideal for use with that sofware, but I'm still trying.
In frigate logs, I notice a lot of errors for each camera connected.
The errors seems caused by the cameras being unable to provide to ffmpeg process of frigate a consistent stream of frames over rtsp.
As a result, I think, the recorded clips in frigate are "incomplete" and the ffmpeg process restart every few minutes for every cameras.
I don't undertsand if this is something unavoidable because of the camera hardware, or if it's caused maybe by the wifi (strenght seems good btw).
I have the feeling that something changed few days ago while at the very beginning I had acceptable results,
Any experince about this setup?
Any settings tweaks in camera and/or frigate that can helps to get best possible results with that hardware?
Thanks

**Comment by roleoroleo** [comment]
You could try to set tcp transport ("-rtsp_transport tcp" whit ffmpeg).
Or you could try to use the alternative rtsp server.
Finally check the wifi connection.

**Comment by uklatt** [comment]
Hi @outofsight,

I tried this hack also with Frigate. It works well when using only the lowres stream (ch0_1.h264) and the alternative RTSP server.
I use model y21ga with firmware 0.2.0.

You can check a recompressed live stream (not Frigate!) at: https://www.twitch.tv/nest_cam

Uwe
